src/features/channels/components/channel-list/search-channel/SearchChannel.tsx

```diff
@@ -97,20 +105,21 @@ const ChannelList = () => {
                       <Box key={channel.channelName}>
```

Comment:
Adding a 'There are no hidden channels' text inside the modal when the array is empty instead of a blank space would be nice

Reply:
Great idea

Reply:
I added it!